www/Gruntfile.js

```diff
@@ -47,28 +52,30 @@ module.exports = function (grunt) {
                     "script/templates/**/*.hbs"
                 ],
                 dest: "generated/precompiled.handlebars.js"
+            },
+            fonts:{
+                src: "fonts/katana_icons/**",
+                dist: "fonts/katana_icons/",
+                config: "fonts/katana_icons/config.json"
             }
         },
-        compass: {
+        fontelloUpdate: {
             options: {
-                sassDir: "<%= files.css.src %>",
-                cssDir: "<%= files.css.dest %>"
-            },
-            prod: {
-                options: {
-                    force: true,
-                    environment: "production",
-                    outputStyle: "nested"
-                }
-            },
-            dev: {
-                options: {
-                    debugInfo: true,
-                    environment: "development",
-                    outputStyle: "nested"
-                }
+                config: "<%= files.fonts.config %>",
+                overwrite: true,
+                fonts: "<%= files.fonts.dist %>"
             }
         },
+        sass: {
+			options: {
```

Comment:
yes, incorrect, but I am not considering this is a major issue and wouldn't fix it